### ISSUE TITLE
Miskos Cave of Chests

### ISSUE THREAD

**Issue by PixelShake92** [MelonSpeedruns/TotkRandomizer#24]
Hi Melon, You likely have moved on, but I just wanted to say how much I appreciated your work on BotW and TotK randomizers. An interesting bug has happened, so I thought I would reach out and share it. The Misko cave of chests in the Eldin region is completely unloading all textures with the exception of a few rocks and a single chest and some glowbulbs. Once again thanks for your work. 

https://github.com/user-attachments/assets/3ce56cb0-f37c-49a6-981b-4fdc51649520

**Comment by MelonSpeedruns** [comment]
Yeah unfortunately I moved on from this, but good to know for other people that attempt to play this. I'm always down for someone to come back to it if they want to, that's why this project is open source!

**Comment by PixelShake92** [comment]
Thanks a lot. I do have one last question for you. How do you interpret the spoiler log files? I tried to see where some items where when I finished a run and it made no sense to me 💀

**Comment by MelonSpeedruns** [comment]
Oh use https://objmap-totk.zeldamods.org/#/ and enter the big number in there to know what chest it is on the map!

**Comment by PixelShake92** [comment]
Ohhh excellent! Thanks a bunch. Btw I've started looking at BotW for
archipelago (have some work started on the wii u version actually) send me
a discord dm if you think you could spare the time to help. (I am aware
archipelago will not allow BotW on their servers yet but didn't stop AM2R
being made).

**Comment by ZobeePlays** [comment]
Hey im super interested in that Archipelago implementation! would love to be kept up with updates on that! Is there a place where that’s being done? (cuz I also know it sure isn’t the main servers lol) Thx!
